Sources/Socket.swift

```diff
@@ -47,37 +47,37 @@ public struct ReceiveMode : OptionSet {
 }
 
 public final class Socket {
-    let socket: UnsafeMutablePointer<Void>
+    let socket: UnsafeMutableRawPointer
 
-    init(socket: UnsafeMutablePointer<Void>) {
+    init(socket: UnsafeMutableRawPointer) {
         self.socket = socket
     }
 
     deinit {
         zmq_close(socket)
     }
 
-    func setOption(_ option: Int32, value: UnsafePointer<Void>?, length: Int) throws {
+    func setOption(_ option: Int32, value: UnsafeRawPointer?, length: Int) throws {
         if zmq_setsockopt(socket, option, value, length) == -1 {
-            throw Error.lastError
+            throw ZeroMqError.lastError
         }
     }
 
-    func getOption(_ option: Int32, value: UnsafeMutablePointer<Void>, length: UnsafeMutablePointer<Int>) throws {
+    func getOption(_ option: Int32, value: UnsafeMutableRawPointer, length: UnsafeMutablePointer<Int>) throws {
         if zmq_getsockopt(socket, option, value, length) == -1 {
-            throw Error.lastError
+            throw ZeroMqError.lastError
         }
     }
 
     public func bind(_ endpoint: String) throws {
         if zmq_bind(socket, endpoint) == -1 {
-            throw Error.lastError
+            throw ZeroMqError.lastError
         }
     }
 
     public func connect(_ endpoint: String) throws {
         if zmq_connect(socket, endpoint) == -1 {
-            throw Error.lastError
+            throw ZeroMqError.lastError
         }
     }
 
```

Comment:
`sendMessage(_:)` to `send(_:)`? That way it fits with the rest of the send methods.